examples/apache2/default.vhost

```diff
@@ -21,7 +21,7 @@ WSGIApplicationGroup %{GLOBAL}
 		# Your LDAP configuration here, including Require directives
 
 		# This user is used by clients to download test cases and signatures
-		Require user fuzzmanager
+		Require valid-user
```

Comment:
If you change this, can you also update the comment above to something like "Users need to be able to download ..."?